.github/workflows/bors.yml

```
@@ -165,7 +165,8 @@ jobs:
           # run this in CI if it gets an incorrect lake hash for existing cache files somehow
           # lake exe cache pack! || true
           lake exe cache commit || true
-          lake exe cache put || true
+          # try twice in case of network errors
+          lake exe cache put || lake exe cache put || true
```

Would it make sense to sleep for a second before we try again? Is this how that is spelled in bash???
```suggestion
          lake exe cache put || (sleep 1; lake exe cache put) || true
```

---

I'm inclined not to. How do we pick 1 second? Will someone later wonder why we picked 1 second and hesitate to just remove this? It just seems unnecessarily complicated.

---

For that we can leave a comment explaining it. I have no idea if it would increase reliability or not, as we have no real way of knowing whats causing these network disconnects. But it seems not harmful to give a little buffer before retrying

---

Okay, I've added these pauses.